### PR TITLE
Add Release Notes for iOS Release 1.69.170

### PR DESCRIPTION
**Pull request by Uni-verse**
Resolves https://github.com/brave/brave-browser/issues/41647

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A
  - [ ] https://github.com/brave/brave-browser/wiki/Brave-Wallet

## Test Plan:
N/A

### CODE REVIEW

File: CHANGELOG_iOS.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+### General
+
+ - Added per-site adblocking in the Brave Shields panel. ([#39573](https://github.com/brave/brave-browser/issues/39573))
+ - Added advanced server selection for Brave VPN. ([#36409](https://github.com/brave/brave-browser/issues/36409))
+ - Added country server details to the VPN “Server Region” page. ([#39824](https://github.com/brave/brave-browser/issues/39824))
+ - Improved standard mode shields logic to allow some ads that were being hidden. ([#40967](https://github.com/brave/brave-browser/issues/40967))
+ - Updated model names in the Leo menu. ([#40464](https://github.com/brave/brave-browser/issues/40464))
+ - Updated adblock-rust to v0.8.11 which applies “removeparam” to document/subdocument/xhr by default. ([#36039](https://github.com/brave/brave-browser/issues/36039))
+ - Updated Leo input box to match recent design changes. ([#38960](https://github.com/brave/brave-browser/issues/38960))
+ - Updated favorite name on New Tab Page from “Twitter” to “X”. ([#39322](https://github.com/brave/brave-browser/issues/39322))
```
Comment (kjozwiak) [inline CHANGELOG_iOS.md:18]: I don't think we need this one in the release notes TBH. Lets just label it `release-notes/exclude` and remove it.

File: CHANGELOG_iOS.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+ - Fixed prompt spoofing issue as reported on HackerOne by Imnarendrabhati. ([#40709](https://github.com/brave/brave-browser/issues/40709))
+ - Fixed sandbox iframe configuration vulnerability as reported on HackerOne by imnarendrabhati. ([#40846](https://github.com/brave/brave-browser/issues/40846))
+ - Fixed issues with Requestblockingscript.js which caused some delays in page loading on certain sites. ([#39546](https://github.com/brave/brave-browser/issues/39546))
+ - Upgraded Chromium to 128.0.6613.138. ([#40972](https://github.com/brave/brave-browser/issues/40972)) ([Changelog for 128.0.6613.138](https://chromium.googlesource.com/chromium/src/+log/128.0.6613.120..128.0.6613.138?pretty=fuller&n=10000))
```
Comment (kjozwiak) [inline CHANGELOG_iOS.md:22]: The chromium changelog link is incorrect. If you take a look at `1.68.145`, it was on `127.0.6533.120` and not `128.0.6613.120`. Should be:

```
 - Upgraded Chromium to 128.0.6613.138. ([#40972](https://github.com/brave/brave-browser/issues/40972)) ([Changelog for 128.0.6613.138](https://chromium.googlesource.com/chromium/src/+log/127.0.6533.120..128.0.6613.138?pretty=fuller&n=10000))
```

File: CHANGELOG_iOS.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+
+### General
+
+ - Added per-site adblocking in the Brave Shields panel. ([#39573](https://github.com/brave/brave-browser/issues/39573))
+ - Added advanced server selection for Brave VPN. ([#36409](https://github.com/brave/brave-browser/issues/36409))
+ - Added country server details to the VPN “Server Region” page. ([#39824](https://github.com/brave/brave-browser/issues/39824))
+ - Improved standard mode shields logic to allow some ads that were being hidden. ([#40967](https://github.com/brave/brave-browser/issues/40967))
+ - Updated model names in the Leo menu. ([#40464](https://github.com/brave/brave-browser/issues/40464))
+ - Updated adblock-rust to v0.8.11 which applies “removeparam” to document/subdocument/xhr by default. ([#36039](https://github.com/brave/brave-browser/issues/36039))
+ - Updated Leo input box to match recent design changes. ([#38960](https://github.com/brave/brave-browser/issues/38960))
```
Comment (kjozwiak) [inline CHANGELOG_iOS.md:17]: Should mention that we're matching a design. Should use something along the lines of the following:

```
- Updated the Leo input box UI. ([#38960](https://github.com/brave/brave-browser/issues/38960))
```

Or something along those lines.

File: CHANGELOG_iOS.md [hunk 4]
```diff
@@ -1,5 +1,26 @@
...
+
+ - Enabled Bitcoin by default in Brave Wallet. ([#38775](https://github.com/brave/brave-browser/issues/38775))
+
+### General
+
+ - Added per-site adblocking in the Brave Shields panel. ([#39573](https://github.com/brave/brave-browser/issues/39573))
+ - Added advanced server selection for Brave VPN. ([#36409](https://github.com/brave/brave-browser/issues/36409))
+ - Added country server details to the VPN “Server Region” page. ([#39824](https://github.com/brave/brave-browser/issues/39824))
+ - Improved standard mode shields logic to allow some ads that were being hidden. ([#40967](https://github.com/brave/brave-browser/issues/40967))
+ - Updated model names in the Leo menu. ([#40464](https://github.com/brave/brave-browser/issues/40464))
```
Comment (kjozwiak) [inline CHANGELOG_iOS.md:15]: Just use the same note that `Desktop` & `Android` used for the same issue:

```
 - Updated model names in Brave Leo models menu. ([#40464](https://github.com/brave/brave-browser/issues/40464))
```

**Review by kjozwiak — CHANGES_REQUESTED** [review]

**Comment by Uni-verse** [comment]
Thanks for the reviews!

**Review by kjozwiak — APPROVED** [review]
LGTM 👍

**Pull request merged**